Makefile

```diff
@@ -160,7 +160,7 @@ bin/subctl-%: generate-embeddedyamls $(shell find pkg/subctl/ -name "*.go") vend
 	GOARCH=$${components[-1]}; \
 	export GOARCH GOOS; \
 	$(SCRIPTS_DIR)/compile.sh \
-		--ldflags "-X github.com/submariner-io/submariner-operator/apis.Version=$(VERSION) \
```

Comment:
This was modified as part of the following PR - https://github.com/submariner-io/submariner-operator/commit/861856b068a130861756f98db029e73cf7eaf10f#diff-8a3b2dde19c3466bd265947fa5bf55af83aa04535ed6ad07654b6221c26bf9fd

I'd like to hear from @skitt about this fix.

Reply:
Thanks @SteveMattar and @sridhargaddam, this fix is correct.